src/Rules/PHPUnit/AttributeRequiresPhpVersionRule.php

```diff
@@ -71,6 +82,29 @@ public function processNode(Node $node, Scope $scope): array
 			if (
 				!is_numeric($args[0])
 			) {
+
+				try {
+					$testPhpVersionConstraint = $parser->parseConstraints($args[0]);
+				} catch (UnexpectedValueException $e) {
+					$errors[] = RuleErrorBuilder::message(
+						sprintf($e->getMessage()),
```

Comment:
not sure we want to 1:1 print the message from composer/semver as a phpstan rule

Reply:
alternative approach could be to somehow analyse `#[RequiresPhp('>=8.0')]` as if it would have the AST of `PHP_VERSION_ID >= 80000` - which would re-use the already existing PHP_VERSION_ID checking capabilities